src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/Wallet.java

```diff
@@ -3135,6 +3080,274 @@ public Call callContractAsync(String contractAddress, String methodName, List<Ca
+    }
+
+    /**
+     * Create a Klaytn fee payer account asynchronously.<br>
+     * Generate a Klaytn account address and random private/public key pair and get ID of public key and private key returned.<br>
+     * Klaytn fee payer account should be updated to AccountKeyRoleBased and can only be used for fee delegation.<br>
+     * POST /v2/feepayer
+     *
+     * <pre>Example
+     * {@code
+     * ApiCallback<Account> callback = new ApiCallback<Account> callback() {
+     *   ....implement callback method.
+     * };
+     *
+     * caver.kas.wallet.createFeePayerAsync(callback);
+     * }
+     * </pre>
+     *
+     * @param callback The callback to handle response.
+     * @return Call
+     * @throws ApiException
+     */
+    public Call createFeePayerAsync(ApiCallback<Account> callback) throws ApiException {
+        return getFeepayerApi().creatFeePayerAccountAsync(chainId, callback);
+    }
+
+    /**
+     * Delete a certain Klaytn fee payer account.<br>
+     * DELETE /v2/feepayer/{address}
+     *
+     * <pre>Example
+     * {@code
+     * String feePayerAddress = "0x{feePayer address}";
+     * AccountStatus status = caver.wallet.kas.deleteFeePayer(feePayerAddress);
+     * }
+     * </pre>
+     *
+     * @param address Klaytn account address.
+     * @return AccountStatus
+     * @throws ApiException
+     */
+    public AccountStatus deleteFeePayer(String address) throws ApiException {
+        return getFeepayerApi().deleteFeePayerAccount(chainId, address);
+    }
+
+    /**
+     * Delete a certain Klaytn fee payer account asynchronously.<br>
+     * DELETE /v2/feepayer/{address}
+     *
+     * <pre> Example
+     * {@code
+     * ApiCallback<AccountStatus> callback = new ApiCallback<AccountStatus> callback() {
+     *   ....implement callback method.
+     * };
+     *
+     * String feePayerAddress = "0x{feePayer address}";
+     * caver.wallet.kas.deleteFeePayerAsync(feePayerAddress, callback);
+     * }
+     * </pre>
+     *
+     * @param address Klaytn account address.
+     * @param callback The callback to handle response.
+     * @return Call
+     * @throws ApiException
+     */
+    public Call deleteFeePayerAsync(String address, ApiCallback<AccountStatus> callback) throws ApiException {
+        return getFeepayerApi().deleteFeePayerAccountAsync(chainId, address, callback);
+    }
+
+    /**
+     * Retrieve a certain Klaytn fee payer account.<br>
+     * GET /v2/feepayer/{address}
+     *
+     * <pre>Example
+     * {@code
+     * String address = "0x{address}";
+     * Account account = caver.kas.wallet.getFeePayer(address);
+     * }
+     * </pre>
+     *
+     * @param address Klaytn account address.
+     * @return Account
+     * @throws ApiException
+     */
+    public Account getFeePayer(String address) throws ApiException {
+        return getFeepayerApi().retrieveFeePayerAccount(chainId, address);
+    }
+
+    /**
+     * Retrieve a certain Klaytn fee payer account.<br>
```

Comment:
```suggestion
     * Retrieve a certain Klaytn fee payer account asynchronously.<br>
```
How about adding `asynchronosuly` at the end of description just like the rest of the comments in the async function?